Project3/index.html

```diff
@@ -0,0 +1,118 @@
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<link rel="stylesheet" href="styles.css">
+<link href="https://fonts.googleapis.com/css?family=BenchNine:300,400,700|Teko:300,400,500,600,700&display=swap"
+    rel="stylesheet">
+<title>Document</title>
+</head>
+
+<body>
+
+    <form action="">
+        <h2>IDENTITY</h2>
+        <label for="real-name">Real Name</label>
+        <input id="real-name" type="text" required>
+
+        <label for="alias"> Alias </label>
+        <input id="alias" type="text" required>
+
+        <label for="email"> Email </label>
+        <input id="email" type="text" required>
+
+        <h2>PERSONAL<br> CHARACTERISTICS</h2>
+
+        <label for="age"> Age </label>
+        <input id="age" type="number" required>
+
+        <fieldset class="left-column">
+            <legend>Category</legend>
+            <p>Please select as many as apply to your origin and how you gained your powers.</p>
+
+            <div>
+                <input name="category" id="alien" type="checkbox">
+                <label for="alien">Alien</label>
+            </div>
+
+            <div><input name="category" id="cyborg" type="checkbox">
+                <label for="cyborg">Cyborg</label></div>
+
+            <div><input name="category" id="experiment" type="checkbox">
+                <label for="experiment">Experiment</label></div>
+
+            <div> <input name="category" id="human" type="checkbox">
+                <label for="human">Human</label></div>
+
+            <div><input name="category" id="inhuman" type="checkbox">
+                <label for="inhuman">Inhuman</label></div>
+
+            <div><input name="category" id="mutant" type="checkbox">
+                <label for="mutant">Mutant</label>
+            </div>
+        </fieldset>
+
+        <fieldset class="right-column">
+            <legend>Gender</legend>
```

Comment:
Per [requirements](https://mica-web.github.io/gd-220/proj3.html#requirements), this field should be **required**.